### PR TITLE
[PT2.0.1][CUDA12] remove ssh keys

### PR DESCRIPTION
**Pull request by jeet4320**
*GitHub Issue #, if available:*

**Note**: 
- If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

- All PR's are checked weekly for staleness. This PR will be closed if not updated in 30 days.

### Tests run

**NOTE: By default, docker builds are disabled. In order to build your container, please update dlc_developer_config.toml and specify the framework to build in "build_frameworks"**
- [x] I have run builds/tests on commit <INSERT COMMIT ID> for my changes.

On commit https://github.com/aws/deep-learning-containers/commit/5459ceb308ca08d3360a672cc591f2e789172661 All tests passed except sanity ECR scan and torchdata.

On commit, torchdata passed https://github.com/aws/deep-learning-containers/pull/3306/commits/3215ddde01a872fff9e0f7a1ec49f675aa02e20b

On commit https://github.com/aws/deep-learning-containers/pull/3306/commits/e573398dd2237b5110aac35060561af118185672 ECR scan passed

**NOTE: If you are creating a PR for a new framework version, please ensure success of the standard, rc, and efa sagemaker remote tests by updating the dlc_developer_config.toml file:**

- [ ] Revision A: `sagemaker_remote_tests = "standard"`
- [ ] Revision B: `sagemaker_remote_tests = "rc"`
- [ ] Revision C: `sagemaker_remote_tests = "efa"`

**Additionally, please run the sagemaker local tests in at least one revision:**
- [ ] `sagemaker_local_tests = true`

### Formatting
- [ ] I have run `black -l 100` on my code (formatting tool: https://black.readthedocs.io/en/stable/getting_started.html)

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON/GRAVITON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true`, `neuron_mode = true` or `graviton_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jeet4320** [comment]
All tests passed except sanity ECR scan.

**Review by ohadkatz — APPROVED** [review]

**Review by ohadkatz — CHANGES_REQUESTED** [review]
Please revert TOML and update branch

**Comment by saimidu** [comment]
The EC2 tests timed out in [5459ceb](https://github.com/aws/deep-learning-containers/pull/3306/commits/5459ceb308ca08d3360a672cc591f2e789172661)

**Comment by saimidu** [comment]
`test_pytorch_training_torchdata_gpu` fails due to `pytest not found`.

**Comment by jeet4320** [comment]
> `test_pytorch_training_torchdata_gpu` fails due to `pytest not found`.

Yes retrying it. Have update PR body with commit and its tests status. There is no change on that test

**Comment by jeet4320** [comment]
> > `test_pytorch_training_torchdata_gpu` fails due to `pytest not found`.
> 
> Yes retrying it. Have update PR body with commit and its tests status. There is no change on that test

still same error, I see we are installing pytest but https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/container_tests/bin/pytorch_tests/testTorchdata#L22

**Comment by jeet4320** [comment]
Getting ECR pull error for 3 continuous runs. Will retry in few hours

File: test/testrunner.py [hunk 1]
```diff
@@ -402,6 +402,7 @@ def main():
                 raise Exception(f"EKS cluster {eks_cluster_name} is not in active state")
 
         # Execute dlc_tests pytest command
+        test_path = os.path.join(test_path, "pytorch/training/test_pytorch_training.py::test_pytorch_training_torchdata_gpu")
         pytest_cmd = ["-s", "-rA", test_path, f"--junitxml={report}", "-n=auto"]
```
Comment (jeet4320) [inline test/testrunner.py:406]: to be reverted

File: pytorch/training/buildspec.yml [hunk 2]
```diff
@@ -84,18 +84,18 @@ images:
...
-    tag_python_version: &TAG_PYTHON_VERSION py310
-    cuda_version: &CUDA_VERSION cu121
-    os_version: &OS_VERSION ubuntu20.04
-    tag: !join [ "2.0.1", "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION,
-                 "-ec2-example" ]
-    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /example, /Dockerfile., *DEVICE_TYPE ]
-    context:
-      <<: *TRAINING_CONTEXT
+  # BuildPyTorchExampleGPUTrainPy3DockerImage:
+  #   <<: *TRAINING_REPOSITORY
```
Comment (saimidu) [inline pytorch/training/buildspec.yml:88]: Will this change be reverted?

**Review by saimidu — COMMENTED** [review]
Approved. Please revert temporary changes.

**Review by saimidu — APPROVED** [review]

**Pull request merged**